src/components/molecules/VenueRoomItem/VenueRoomItem.tsx

```diff
@@ -80,7 +80,10 @@ export const VenueRoomItem: React.FC<VenueRoomItemProps> = ({
     // TS doesn't work properly with const statements and won't 'know' that this is already checked.
     // That's why this is inline instead of isVenuePortal
     if (template !== RoomTemplate.external) {
```

Comment:
You can turn `isVenuePortal` to a [type-guard function](https://basarat.gitbook.io/typescript/type-system/typeguard#user-defined-type-guards) and get rid of the duplication & comment above

Reply:
This is going to be removed in: https://github.com/sparkletown/sparkle/pull/2354/files